tests/integration/google_cloud_storage/test_google_cloud_storage.py

```diff
@@ -60,34 +58,39 @@ def google_cloud_storage_configured():
 )
 @pytest.mark.parametrize("use_datastore_profile", [False, True])
 class TestGoogleCloudStorage:
-    #  must be static, in order to get access from setup_class or teardown_class.
-    @staticmethod
-    def clean_test_directory(bucket_name, object_dir, gcs_fs):
-        test_dir = f"{bucket_name}/{object_dir}/"
-        if gcs_fs.exists(test_dir):
-            gcs_fs.delete(test_dir, recursive=True)
+    assets_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), "assets")
+    _bucket_name = config["env"].get("bucket_name")
+    test_dir = "test_mlrun_gcs_objects"
+    run_dir = f"{test_dir}/run_{uuid.uuid4()}"
+    profile_name = "gcs_profile"
+    credentials_path = config["env"].get("credentials_json_file")
+    test_file = os.path.join(assets_path, "test.txt")
```

Comment:
I'm not really sure I understand why in some places it's `os.path.join`, and some places it's just `f"{...}/{...}"`.

Reply:
os.path.join for local file system join (should work on windows too).

{...}/{...} for remote path join.